Lib/idlelib/outwin.py

```diff
@@ -1,43 +1,115 @@
+"""Editor window that can serve as an output file.
+"""
+
 import re
 
-from tkinter import *
-import tkinter.messagebox as tkMessageBox
+from tkinter import messagebox
 
 from idlelib.editor import EditorWindow
 from idlelib import iomenu
 
 
-class OutputWindow(EditorWindow):
+file_line_pats = [
```

Comment:
Why move out the internal parts? We can access these by `OutputWindow.file_line_pats`

Reply:
I agree that it would have been better in certain respects to start with a pure docstring patch, then tests of code as is, then a code fix-up and movement patch.  This is more or less what we did with configdialog.  This would be at least 2, maybe 3 issues.  However, I have reviewed it as it is.